netmanager/src/views/components/DataDisplay/DeviceView/DeviceLogs.js

```
@@ -99,6 +100,11 @@
       description: description
     };
 
+    // Add user_id only if it exists
+    if (currentUser && currentUser._id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)